sniplates/templatetags/sniplates.py

```diff
@@ -308,17 +308,22 @@ def form_field(context, field, widget=None, **kwargs):
         field_data[attr] = getattr(field.field, attr, None)
 
     # Normalize the value [django.forms.widgets.Select.render_options]
-    value = field.value()
-    if value is None:
-        pass
-    elif isinstance(value, (list, tuple)):
-        value = map(force_text, value)
+    _value = field.value()
+    if _value is None:
+        value = _value
+    elif isinstance(_value, (list, tuple)):
+        value = map(force_text, _value)
     else:
-        value = force_text(field_data['value']())
+        value = force_text(_value)
     field_data['value'] = value
 
     if field_data['choices']:
-        field_data['display'] = dict(field.field.choices).get(field_data['value'], '')
+        _choices = dict(field.field.choices)
+        if isinstance(_value, (list, tuple)):
+            bits = [_choices.get(val, '') for val in _value]
+            field_data['display'] = ', '.join(bits)
```

Comment:
Yeah, in playing around with it this morning I've come to the conclusion 'display' doesn't make sense in a non-scalar context... it's there to make it easy for you to show the matching 'display' string for the 'value' of the field.

If someone can come up with an argument for needing it, I can work in support for value->display mapping [probably a filter] ... otherwise, I'm only going to set display for single values.